source/DailyOverview/DailyOverview.html

```diff
@@ -10,8 +10,13 @@
     <body>
         <div id="header">
             <h1 id="monthView">
-                <a href="../MonthlyOverview/MonthlyOverview.html" id="words">
-                </a>
+                <a
+                    href="http://127.0.0.1:5500/source/MonthlyOverview/MonthlyOverview.html"

```

Comment:
I feel like we should keep the ".." notation as we would have to change the link for the actual version since the hosted version can't access http://127.0.0.1:5500